### PR TITLE
14-Uploading custom css for the site

### PR DESCRIPTION
**Pull request by godfreym**
## Note:
The admin of the site would like to be able to customize the `CSS` and change some of the elements styles like colours, font weight etc
## Story:
**AS An Admin** of the site
**I WANT TO** be able to upload my custom `CSS`
**SO THAT** I can customize the elements of the site when I need to do so

## Acceptance Criteria:
- [x] GIVEN I am the admin WHEN I want to customize the site with custom css THEN I should be able to upload the custom css file to a site

### CODE REVIEW

**Comment by godfreym** [comment]
@leonardk1 Please Review

**Comment by godfreym** [comment]
@leonardk1 Please Review

**Comment by leonardk1** [comment]
@godfreym does this allow the admin to see changes only after uploading, or is there a way to see the changes before

**Comment by leonardk1** [comment]
@godfreym the code looks good?

**Comment by godfreym** [comment]
@leonardk1 That will come later not in this PR, This deals with uploading css files

**Comment by godfreym** [comment]
@arafatm Please Review

**Comment by leonardk1** [comment]
@godfreym :+1:

**Comment by godfreym** [comment]
@arafatm Please Review

File: db/migrate/20151214060428_add_css_to_companies.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddCssToCompanies < ActiveRecord::Migration
+  def change
+    add_column :companies, :css, :string
```
Comment (arafatm) [inline db/migrate/20151214060428_add_css_to_companies.rb]: @godfreym you're storing the custom css in the database?

**Comment by arafatm** [comment]
@godfreym go ahead and merge. not sure storing the css in the database is the right approach. let's discuss tomorrow after demoing to jim.

File: db/migrate/20151214060428_add_css_to_companies.rb (hunk 1 above)
Comment (godfreym) [inline db/migrate/20151214060428_add_css_to_companies.rb]: @arafatm NO, the css file is stored in `app/assets/stylesheets` directory... this attribute just stores the name of the file which links to the original file on asset directory

Comment (arafatm) [inline db/migrate/20151214060428_add_css_to_companies.rb]: Nice. Carry on.

**Comment by godfreym** [comment]
@arafatm I am not storing the css in the database, when you upload the file then I have directed the css file to go to stylesheets directory but this `css` attribute you see on the database company table just helps to track the name of the css file to link to for the company since we expect each company to have its own custom css file with its own naming for the file so later we can load the file within `<head>` tag as:
 `<% link_for_custom_theme(load_css_from_app/assets/stylesheets(filename: @company.css )) %>`

**Pull request merged**